riff-raff/public/docs/howto/fix-a-failed-deploy.md

```diff
@@ -9,32 +11,18 @@ the desired capacity stays the _same_ as the specified maximum capacity for the
 state, Riff-Raff/Magenta will not be able to do any more automated deploys, because it can't double
```

Comment:
Minor: there are a couple of references to `Riff-Raff/Magenta` in this section. I'm not sure many people know (or need to know!) about Magenta's existence, so I'd be tempted to shorten these to `Riff-Raff`!

Reply:
Yes, I agree! Thank you @jacobwinch! 👍🏽☺️